src/stateManagement/textStyle.js

```diff
@@ -1,38 +1,56 @@
-let defaultFontSize = 15,
-  defaultFont = `${defaultFontSize}px Arial`,
-  defaultBackgroundColor = 'transparent';
+const textStyle = {
+  fontSize: 15,
+  fontFamily: 'Arial',
+  backgroundColor: 'transparent',
+};
 
+// Deprecate
 function setFont(font) {
-  defaultFont = font;
+  const split = font.split('px ');
+
+  if (split.length === 2) {
+    setFontSize(split[0]);
+    setFontFamily(split[1]);
+  }
 }
 
 function getFont() {
-  return defaultFont;
+  return `${textStyle.fontSize}px ${textStyle.fontFamily}`;
+}
+
+function setFontFamily(fontFamily) {
+  textStyle.fontFamily = fontFamily;
+}
+
+function getFontFamily() {
+  return textStyle.fontFamily;
 }
 
 function setFontSize(fontSize) {
-  defaultFontSize = fontSize;
+  textStyle.fontSize = parseFloat(fontSize);
 }
 
 function getFontSize() {
-  return defaultFontSize;
+  return textStyle.fontSize;
 }
 
 function setBackgroundColor(backgroundColor) {
-  defaultBackgroundColor = backgroundColor;
+  textStyle.backgroundColor = backgroundColor;
```

Comment:
as far as default backgroundColor has value equals 'transparent' its ok to not do any kind of validation, but I would recommend to at least add a comment here

Reply:
This functions accepts any value that represents a color. Examples:

- transparent
- red, green, blue
- rgb(255,0,0)
- rgba(0,255,0,0.5)
- #FFC
- #FF0000
- #FF993399
- And any format accepted by HTML

Reply:
JSDoc will be added to the function

Reply:
JSDoc is not updated yet